com.microsoft.mrtk.input/Visualizers/ControllerVisualizer/ControllerModelLoader.cs

```diff
@@ -66,7 +66,7 @@ public async static Task<GameObject> TryGenerateControllerModelFromPlatformSDK(I
             {
                 if(!warningCache.TryGetValue(inputDevice, out bool warningLogged) || !warningLogged)
                 {
-                    Debug.LogWarning("Failed to obtain controller model key from platform.");
+                    Debug.LogFormat("{0} didn't provide a key for a controller model from the platform.", inputDevice.name);
```

Comment:
Style nit, c# native string interpolation probably preferred to LogFormat but that's fine